test/unit/mongo_logger.test.ts

```diff
@@ -1612,3 +1624,127 @@ describe('class MongoLogger', function () {
     }
   });
 });
+
+describe('stringifyWithMaxLen', function () {
+  let returnVal: string;
```

Comment:
Doesn't appear to be used outside each block, what's this hoisted for?

Reply:
Good catch, cruft from previous iterations of the tests.